powder_reduction.py

```diff
@@ -186,7 +193,7 @@ def powder_reduction(sample='sample.nxs',
     # 4. Focus panels
     # Assuming sample is in d-spacing: Focus into groups
 
-    focused = sc.groupby(sample_dspacing, group='group').flatten('spectrum')
+    focused = sc.groupby(sample_dspacing.unaligned, group='group').flatten('spectrum')
```

Comment:
Does
```python
focused = sc.groupby(sample_dspacing, group='group').sum('spectrum')
```
work here? It should in principle, but I have not tested this in all cases yet, so I cannot exclude that it does not.

Reply:
It works without "unaligned". But there is no data.

Reply:
Correct, `focused` would also be realigned data in this case, but that is what you need, I believe?

Reply:
You are correct. At this final stage, it is what I need.